inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/interpolate.cpp

```diff
@@ -28,28 +28,27 @@ const std::vector<std::set<size_t>> axes = {
         {2, 3},
 };
```

Comment:
"axes" is not used in test instance. if axis is not defined, the default is [0,1,2,3] is uesd for 4 rank tensor, then we need specify targetShapes with full size [1,4, 40,40].

Reply:
Fixed.